lib/customers.js

```diff
@@ -281,6 +281,20 @@ function enhanceAtFields (fields) {
   return _.merge(fields, atFields)
 }
 
+function updateRawData (fields) {
+  const fieldsWithRawData = [
+    'id_card_data'
+  ]
+
+  const fieldsToUpdate = _.intersection(fieldsWithRawData, _.keys(fields))
+  const rawFields = _.fromPairs(_.map(f => [`${f}_raw`, JSON.stringify(fields[f].raw)], fieldsToUpdate))
+  const fieldsWithoutRawData = _.mapValues(f => {
+    return _.has(f, 'raw') ? _.omit(['raw'], f) : f
+  }, fields)
```

Comment:
this still has the potential of breaking fields not on fieldsWithRawData. What about something like:

```js
const fieldsWithRawData = [
  'id_card_data'
]

const mapPairToRaw = _.map(([key, value]) => ([
  [key, _.omit(['raw'], value)],
  [`${key}_raw`, JSON.stringify(value.raw)]
]))

function updateRawData (fields) {
  const pickTransformable = _.pick(fieldsWithRawData)
  const extractRaws = _.compose(_.fromPairs, _.flatten, mapPairToRaw, _.toPairs)
  const assignOriginal = _.assign(fields)

  return _compose(assignOriginal, extractRaws, pickTransformable)(fields)
}
```